repos/spack_repo/builtin/packages/acts/package.py

```diff
@@ -423,18 +435,24 @@ class Acts(CMakePackage, CudaPackage):
     depends_on("python", when="+python")
     depends_on("python@3.8:", when="+python @19.11:19")
     depends_on("python@3.8:", when="+python @21:")
```

Comment:
https://github.com/acts-project/acts/blob/7b1edddb4dad8db5a5ef5beb39a9dc68fbc1d704/codegen/src/codegen/sympy_common.py#L281

The sympy codegen includes nested f-strings with quoting only supported in python 3.12 and later.